### PR TITLE
Add ability to run a query from log group window

### PR DESCRIPTION
**Pull request by kiiadi**
Screen Shot 2020-12-01 at 2 10 18 PM

Also remove "open query editor" button and move it to a toolbar on the right so its more consistent with other toolwindows
Screen Shot 2020-12-01 at 2 10 47 PM

### CODE REVIEW

**Review by hunterwerlla — APPROVED** [review]

**Review by abrooksv — APPROVED** [review]

File: jetbrains-core/resources/telemetryOverride.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
-    "types": [],
+    "types": [
+        {
```
Comment (rli) [inline jetbrains-core/resources/telemetryOverride.json:3]: probably fix and move into common

Comment (kiiadi) [reply jetbrains-core/resources/telemetryOverride.json:3]: https://github.com/aws/aws-toolkit-common/pull/198

Comment (kiiadi) [reply jetbrains-core/resources/telemetryOverride.json]: done

**Pull request merged**